src/vivarium_public_health/mslt/delay.py

```diff
@@ -7,11 +7,17 @@
 lifetable simulation.
 
 """
+from typing import Any, Dict, List, Optional
+
 import numpy as np
 import pandas as pd
+from vivarium import Component
+from vivarium.framework.engine import Builder
+from vivarium.framework.event import Event
+from vivarium.framework.population import SimulantData
 
 
-class DelayedRisk:
+class DelayedRisk(Component):
```

Comment:
What is the reasoning for making this a subclass of Component and not the VPH risk class?

Reply:
It's an entirely different way of modelling risks that doesn't actually bear any resemblance to the standard VPH Risk component.